egs2/iemocap/asr1/local/lowercase.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env perl
+#
+# This file is part of moses.  Its use is licensed under the GNU Lesser General
+# Public License version 2.1 or, at your option, any later version.
+# Modified for espnet2 iemocap recipe
+# Yushi Ueda, Carnegie Mellon University (2021)
+
+use warnings;
+use strict;
+
+while (@ARGV) {
```

Comment:
We have `moses` installation available at espnet, maybe we could use that directly?

Reply:
Thanks, but I slightly modified the code to fit to this recipe, so we couldn't use moses directly in this case.